home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/RSQLUserIT.java

```diff
@@ -0,0 +1,267 @@
+package com.softserveinc.ita.homeproject.api.tests;
+
+import com.softserveinc.ita.homeproject.ApiException;
+import com.softserveinc.ita.homeproject.api.UserApi;
+import com.softserveinc.ita.homeproject.model.CreateUser;
+import com.softserveinc.ita.homeproject.model.ReadUser;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RSQLUserIT {
```

Comment:
I expect comments to be the same a for the previous test. Please fix them also for this one.